### ISSUE TITLE
!important are not managed correctly

### ISSUE THREAD

**Issue by JJK801** [finkef/react-native-tailwind.macro#11]
Hi,

I'm not sure this bug is related to this lib or tailwind-react-native-classnames but using `!` (to make CCS rule as important) prefix seems not working.

## Use case:

I need to hide elements for small screens

- When using no breakpoint:  `display: none` is defined in the element style
- When using a breakpoint:  `display: none` is defined as `!important` in the media query 

so i need anyway to make the override `!important` in the media query

## What i tried:

- use prefix directly in style (`hidden md:!flex`) => Do not work, the `!important` flag is not added to the MQ
- create a dedicated utility:
```javascript
const plugin = require('tailwindcss/plugin')

{
...
plugins: [
    plugin(({ addUtilities }) => {
      addUtilities({
        show: {
          display: 'flex !important',
        },
      })
    }),
  ],
```
this generates a bad formatted CSS Rule (`display: flex !important !important`), since it adds an `!important` flag when it detect a `! ` char (but it doesn't remove it), i also tried with `!flex` / `flex !`, but it's the same result (adds the flag but keeps the `!` char making the rule invalid

## Workaround

I've found a Q&D workaround to make it works on web: `display: 'flex /* ! */` (which results `display: flex /* ! */ !important`), but while it's working on web and was working on our previous native releases, it breaks our last native versions (upgraded RN and using hermes engine)

**Comment by finkef** [comment]
When using a breakpoint, we automatically add the `!important` to make sure all the appropriate styles are applied regardless of order.

In the default case however, we're 1:1 using the styles that twrnc give us, so I think you should raise this issue over there.

I'm curious though, wouldn't a style like `tw="hidden sm:flex"` solve this without using `!important`?

**Comment by JJK801** [comment]
@finkef thanks for you're response.

That's what i expected, but in fact the `!important` flag is added when i use `sm:hidden` (for example) but not with `sm:flex` (maybe it's a bug with this specific behavior 🤷 )

**Comment by JJK801** [comment]
Here is some captures of what is generated with `hidden desktop:flex` (`desktop` is a custom breakpoint, but it behaves the same with defaults)

Styles:

![Capture d’écran 2022-05-10 à 10 28 16](https://user-images.githubusercontent.com/2821990/167584937-519b2e1c-3c23-4543-8fb2-b468d978f17c.png)

Media Query:

![Capture d’écran 2022-05-10 à 10 31 05](https://user-images.githubusercontent.com/2821990/167585136-476d0eea-f88b-4acf-9cc8-5b5370a4ab14.png)

**Comment by finkef** [comment]
Got it, it was missing `!important` for vendor-prefixed styles. Fixed in #13, it should now work with version 1.3.1 from npm.

Closing, feel free to reopen if the issue persists! 🙌

**Comment by JJK801** [comment]
Thanks 👍
